state-sync/storage-service/server/src/handler.rs

```diff
@@ -246,8 +247,8 @@ impl<T: StorageReaderInterface> Handler<T> {
             .is_some()
         {
             sample!(
-                SampleRate::Duration(Duration::from_secs(ERROR_LOG_FREQUENCY_SECS)),
-                warn!(LogSchema::new(LogEntry::OptimisticFetchRequest)
+                SampleRate::Duration(Duration::from_secs(INFO_LOG_FREQUENCY_SECS)),
+                info!(LogSchema::new(LogEntry::OptimisticFetchRequest)
```

Comment:
We can probably just remove this log line, or keep it and downgrade to `debug`. 😄